### PR TITLE
WIP Wye tech news 3

### PR DESCRIPTION
**Pull request by spastorino**

### CODE REVIEW

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+<!--more-->
+
+
+**Ruby 2.3.0-preview2 Released**<br/>
+https://www.ruby-lang.org/en/news/2015/12/11/ruby-2-3-0-preview2-released/<br/>
+The most remarkable changes since preview1 are those to support a pre-compilation system and a few performance tuning fixes.
+
+**Ruby CVE-2015-7551: Unsafe tainted string usage in Fiddle and DL**<br/>
+https://www.ruby-lang.org/en/news/2015/12/16/unsafe-tainted-string-usage-in-fiddle-and-dl-cve-2015-7551/<br/>
+There is an unsafe tainted string usage vulnerability in Fiddle and DL. This vulnerability has been assigned the CVE identifier CVE-2015-7551.<br/>
```
Comment (spastorino) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: <br/> not needed

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+Ruby 2.2.4, Ruby 2.1.8 and Ruby 2.0.0-p648 were released including a security fix for Fiddle extension.<br/>
+https://www.ruby-lang.org/en/news/2015/12/16/ruby-2-2-4-released/<br/>
+https://www.ruby-lang.org/en/news/2015/12/16/ruby-2-1-8-released/<br/>
+https://www.ruby-lang.org/en/news/2015/12/16/ruby-2-0-0-p648-released/<br/>
```
Comment (spastorino) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: `<br/>`

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+**Rails 5.0.0.beta1: Action Cable, API mode, Rails command**<br/>
+http://weblog.rubyonrails.org/2015/12/18/Rails-5-0-beta1/<br/>
+The first beta release of the new major version of Rails was released.<br/>
```
Comment (spastorino) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: the/this/

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+In addition, you might be interested on how to upgrade a Rails app to
```
Comment (spastorino) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: interested in

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+Rails 5:<br/>
+http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-4-2-to-rails-5-0
+
+**Rails 5: Action Cable demo**<br/>
+https://www.youtube.com/watch?v=n0WUjGkDFS0&feature=youtu.be<br/>
+A quick demo of Action Cable in Rails 5 by building a simple chat application.
```
Comment (spastorino) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: `A quick demo of how to build a Rails 5 simple chat application using Action Cable.`

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+https://github.com/rails/rails/blob/8c752c7ac739d5a86d4136ab1e9d0142c4041e58/activerecord/lib/active_record/attributes.rb<br/>
+A new `attribute` class method is available in Rails 5 models, allowing you to easily define a relationship between the model and a non-Active Record type.
+
+**ApplicationRecord**<br/>
+https://github.com/rails/rails/pull/22567<br/>
+`ApplicationRecord` model superclass in Rails 5
+
+**ActiveRecord::Relation#or**<br/>
+https://github.com/rails/rails/pull/16052<br/>
+`ActiveRecord::Relation` has a new `#or` in Rails 5.
```
Comment (spastorino) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: `#or method` ?

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+https://www.youtube.com/watch?v=n0WUjGkDFS0&feature=youtu.be<br/>
+A quick demo of Action Cable in Rails 5 by building a simple chat application.
+
+**Rails Attributes API**<br/>
+https://github.com/rails/rails/blob/8c752c7ac739d5a86d4136ab1e9d0142c4041e58/activerecord/lib/active_record/attributes.rb<br/>
+A new `attribute` class method is available in Rails 5 models, allowing you to easily define a relationship between the model and a non-Active Record type.
+
+**ApplicationRecord**<br/>
+https://github.com/rails/rails/pull/22567<br/>
+`ApplicationRecord` model superclass in Rails 5
```
Comment (spastorino) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: All models inherit from `ApplicationRecord` in Rails 5, this means no more monkey patches of `ActiveRecord::Base`.

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+author:
+  name: Jorge Bejar
+  email: jorge@wyeworks.com
+  twitter_handle: jmbejar
+  github_handle:  jmbejar
+  image:  /images/team/jorge-bejar.jpg
+  description: Software Engineer at Wyeworks. Ruby on Rails developer.
+---
+
+Hey it's Monday! So here is a new issue of our weekly compilation of interesting news and articles about Ruby, Rails, JavaScript and web development related stuff.
```
Comment (rodrigopdl) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: What about rephrasing it to something along these lines? "Hey! It's Monday! You know what that means? It's time for a new issue of..."

File: source/_posts/2015-12-21-wye-tech-news-3.markdown (hunk 4 above)
Comment (rodrigopdl) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: What do you think of "interested in upgrading a Rails app"?

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+
+Hey! It's Monday! You know what that means? It's time for a new issue of our weekly compilation of interesting news and articles about Ruby, Rails, JavaScript and web development related stuff.
+
+<!--more-->
+
+**Ruby CVE-2015-7551: Unsafe tainted string usage in Fiddle and DL**<br/>
+https://www.ruby-lang.org/en/news/2015/12/16/unsafe-tainted-string-usage-in-fiddle-and-dl-cve-2015-7551/<br/>
+There is an unsafe tainted string usage vulnerability in Fiddle and DL. This vulnerability has been assigned the CVE identifier CVE-2015-7551.
+
+Ruby 2.2.4, Ruby 2.1.8 and Ruby 2.0.0-p648 were released including a security fix for Fiddle extension.<br/>
```
Comment (spastorino) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: Bold this ^ I guess

Comment (spastorino) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: Or is it like a sub-item?

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+https://www.ruby-lang.org/en/news/2015/12/16/ruby-2-2-4-released/<br/>
+https://www.ruby-lang.org/en/news/2015/12/16/ruby-2-1-8-released/<br/>
+https://www.ruby-lang.org/en/news/2015/12/16/ruby-2-0-0-p648-released/
+
+**Rails 5.0.0.beta1: Action Cable, API mode, Rails command**<br/>
+http://weblog.rubyonrails.org/2015/12/18/Rails-5-0-beta1/<br/>
+The first beta release of this new major version of Rails.
+
+In addition, you might be interested in upgrading a Rails app to Rails 5:<br/>
```
Comment (spastorino) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: Same happens here ^

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+https://github.com/rails/rails/pull/16052<br/>
+`ActiveRecord::Relation` has a new `#or` method in Rails 5.
+
+**New Command Router**<br/>
+https://github.com/rails/rails/issues/18878<br/>
+Starting in Rails 5, many of these old `rake` commands can be run with `rails` instead.
+
+**Inspect network traffic in Capybara with Poltergeist**<br/>
+https://medium.com/raise-coffee/inspect-network-traffic-in-capybara-with-poltergeist-16363ec8ecf4<br/>
+Get all the informations you would see in the Network-Tab of your browser if you're using the Poltergeist driver in your Capybara tests.
```
Comment (rodrigopdl) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: informations -> information

"in the Network-Tab of your browser" -> "in your browser's Network-Tab when using Poltergeist driver in your ...."

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+
+**Avoiding "Call Super" with Callbacks**<br/>
+http://weblog.jamisbuck.org/2015/12/19/avoiding-call-super-with-callbacks.html<br/>
+How to use ActiveSupport::Callbacks to avoid the Call Super anti-pattern
```
Comment (rodrigopdl) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: Missing "." at the end.

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+
+**Ruby Refinements**<br/>
+http://jakeyesbeck.com/2015/12/13/ruby-refinements/<br/>
+Refinements are context specific alterations to a class' methods.
```
Comment (spastorino) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: I'm not sure I'd add this one and given that you already had some doubts about it ...

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+http://nithinbekal.com/posts/distributed-ruby/<br/>
+The Ruby stdlib contains a little known library called `dRuby`, which allows multiple Ruby processes to talk to each other over the network.
+
+**Introducing Background Sync**<br/>
+https://developers.google.com/web/updates/2015/12/background-sync?hl=en<br/>
+Background sync is a new web API that lets you defer actions until the user has stable connectivity. This is useful for ensuring that whatever the user wants to send, is actually sent.
+
+**Compiling to WebAssembly: It's Happening!**<br/>
+https://hacks.mozilla.org/2015/12/compiling-to-webassembly-its-happening/<br/>
+WebAssembly is a new binary format for compilation to the web that is in the process of being designed and implemented.
```
Comment (rodrigopdl) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: "that is" -> "that's"

File: source/_posts/2015-12-21-wye-tech-news-3.markdown [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+http://weblog.jamisbuck.org/2015/12/19/avoiding-call-super-with-callbacks.html<br/>
+How to use ActiveSupport::Callbacks to avoid the Call Super anti-pattern
+
+**Ruby Refinements**<br/>
+http://jakeyesbeck.com/2015/12/13/ruby-refinements/<br/>
+Refinements are context specific alterations to a class' methods.
+
+**Ruby Can Be Faster with a Bit of Rust**<br/>
+http://www.sitepoint.com/ruby-can-be-faster-with-a-bit-of-rust/<br/>
+An introduction to Rust for Rubyists, showing how to write a dynamic library to allow Ruby to consume Rust, making Ruby faster when it's done right.
```
Comment (rodrigopdl) [inline source/_posts/2015-12-21-wye-tech-news-3.markdown]: "when it's done right." -> "when done right."

**Pull request merged**